src/components/GraphFilter/GraphFind.tsx

```diff
@@ -100,10 +166,25 @@ export class GraphFind extends React.PureComponent<GraphFindProps> {
     this.handleFind();
   };
 
-  private handleFindSubmit = event => {
-    event.preventDefault();
-    this.findValue = this.findInputValue;
-    this.handleFind();
+  private handleFilter = () => {
+    if (!this.props.cyData) {
+      console.debug('Skip Filter: cy not set.');
+      return;
+    }
+    const cy = this.props.cyData.cyRef;
+    const selector = this.parseFindValue(this.filterValue);
+    cy.startBatch();
+    // restore old filter-hits
+    if (this.filteredElements) {
+      this.filteredElements.restore();
+      this.filteredElements = undefined;
+      runLayout(cy, this.props.layout);
```

Comment:
Mmh, maybe you don't need this anymore if you use `this.filteredElements.style({display: 'element'});`